### PR TITLE
Create lint.yml

### PR DESCRIPTION
**Pull request by git-bd1599**
ADD Github Action to automate linting - lint.yml

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      # ESLint and Prettier must be in `package.json`
+      - name: Install Node.js dependencies
+        run: npm ci
```
Comment (PIG208) [inline .github/workflows/lint.yml:29]: We are using yarn for package management. You can check [this](https://github.com/AnubisLMS/Anubis/blob/master/.github/workflows/test-web.yml#L33) out

**Review by PIG208 — COMMENTED** [review]
Just took a look at [wearerequired/lint-action](https://github.com/wearerequired/lint-action). It seems that they produce some pretty nice-looking outputs. Never tried the auto-fix feature tho. A good thing about having a separate workflow for linting is that other parts of the ci won't be interrupted. I'm feeling that this might also be a nice thing to have for the API to enforce linting.

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -23,6 +26,8 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: 12
+      - run: yarn
+      - run: yarn run run-linters
```
Comment (PIG208) [inline .github/workflows/lint.yml:30]: We shouldn't use `yarn run run-linters` here, and `run-linters` isn't known to yarn. We are already running the linters with `lint-action` below.

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -32,4 +37,4 @@ jobs:
         uses: wearerequired/lint-action@v1
         with:
           eslint: true
-          prettier: true
+      # prettier: true
```
Comment (PIG208) [inline .github/workflows/lint.yml:40]: Instead of commenting out, if we don't need it, just delete it.

**Review by PIG208 — COMMENTED** [review]
LGTM overall. Might need a little bit further discussion on whether we need prettier or not as I'm not sure what's @nysteo plan for frontend linting. As a side note, I would squash these three commits into one, but this is totally up to you as we squash the PR when it's merged anyway.

File: .github/workflows/lint.yml (hunk 3 above)
Comment (nysteo) [reply .github/workflows/lint.yml:40]: @git-bd1599 please fix

File: .github/workflows/lint.yml (hunk 2 above)
Comment (nysteo) [reply .github/workflows/lint.yml:30]: @git-bd1599 please fix

File: .github/workflows/lint.yml (hunk 1 above)
Comment (git-bd1599) [reply .github/workflows/lint.yml:33]: Modified as suggested

File: .github/workflows/lint.yml (hunk 2 above)
Comment (git-bd1599) [reply .github/workflows/lint.yml]: fixed

File: .github/workflows/lint.yml (hunk 3 above)
Comment (git-bd1599) [reply .github/workflows/lint.yml]: Have deleted prettier for now.

**Pull request closed without merging**